change/@ni-eslint-config-angular-2b894e10-2f6e-4ff2-ade2-fae0335c38d8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Some manual testing that I did:
1. verified that introducing a lint error in each of our test files still results in failing that test 
2. verified that vscode lint annotations and fix commands still work (required restarting vscode after switching to this branch and installing dependencies) 

👍